developer_tools/update_introspection_data

```diff
@@ -134,8 +155,12 @@ def main():
     if return_code != 0:
         exit(return_msg)
 
+    # call GetManagedObjects() here, and find the pool_object_path and dev_object_paths
```

Comment:
This is correct. So here you want to use the ObjectManager class that you've built above.

```
managed_objects = ObjectManager.Methods.GetManagedObjects(proxy, {})
```

Then you want to use the ```pools()``` function. Something like here:
https://github.com/stratis-storage/stratis-cli/blob/develop-2.2.1/src/stratis_cli/_actions/_top.py#L310.

Reply:
I tried this, as a test (setting a "pool_name" variable, since it's otherwise undefined right now):
```
    pool_name = "pool_name"

    managed_objects = ObjectManager.Methods.GetManagedObjects(proxy, {})
    (pool_object_path, _) = next(
        pools(props={"Name": pool_name})
        .require_unique_match(True)
        .search(managed_objects)
    )
```
...and I get this: 
```
dbus_client_gen._errors.DbusClientUnknownSearchPropertiesError: These properties in the specified query are unknown to interface "org.storage.stratis2.pool.r1": Name
```

Reply:
Oh, that's right; I'm operating with "a minimal chunk of introspection data", so I need to add the new property that I'm using:
```
<property access="read" name="Name" type="s">
</property>
```

Now execution seems to continue as before (but I still need to do something.)

Reply:
I need to populate dev_object_paths now.  However, I did a "before and after" comparison, to confirm that what I did above is making some progress:

Before:
```
(Pdb) pool_object_path
dbus.ObjectPath('/')
```

After:
```
(Pdb) pool_object_path
dbus.ObjectPath('/org/storage/stratis2/1')
```